docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1911 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        and they will be treated seperately (returned as an additional output) in the parent function <code>_murlin</code>.
</pre><p>I think we should remove this note</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Correct. Will be fixed next time.</p>